open-company/open-company-web
# Devcards fix

card: https://trello.com/c/EckUGxnz

To test:
- `lein clean && lein devcards!`
- visit [/devcards](http://localhost:3449/devcards)
- check that the devcards work
